DeskAPIClient/DeskAPIClient/DSAPIResource.m

```diff
@@ -572,7 +572,7 @@ - (NSURLSessionDataTask *)listResourcesForRelation:(NSString *)relation
                                        notModified:(DSAPIPageSuccessBlock)notModified
                                            failure:(DSAPIFailureBlock)failure
 {
-    DSAPILink *linkToRelation = [self linkForRelation:relation];
+    DSAPILink *linkToRelation = [self linkForRelation:relation]; //TODO: add nil check and return internal error if link is nil
```

Comment:
@nartiles I created this ticket https://jira.desk.technology/browse/APIC-157 to address this issue.